doc/changelog.rst

```diff
@@ -38,6 +63,11 @@ Unavoidable breaking changes
   Users of the Stable API with estimated_document_count are recommended to upgrade
   their server version to 5.0.9+ or set :attr:`pymongo.server_api.ServerApi.strict`
   to ``False`` to avoid encountering errors (`PYTHON-3167`_).
+- Removed generic typing from :class:`~pymongo.client_session.ClientSession` to improve
+  support for pyright (`PYTHON-3283`_).
+- Added ``__all__`` to the bson, pymongo, and gridfs packages. This could be a breaking
```

Comment:
It seems that it is the minority in the change log to not capitalize PyMongo:
```
# Filter out the instances where it ends in a period, because most of the time that's a :class:
➜  mongo-python-driver git:(master) rg -c "pymongo[^[.]]" ./doc/changelog.rst
20
➜  mongo-python-driver git:(master) rg -c "PyMongo" ./doc/changelog.rst
204
```

Reply:
This is refering the the package name which is "pymongo" as in "import pymongo".